nav2_recoveries/plugins/back_up.hpp

```diff
@@ -51,6 +51,7 @@ class BackUp : public Recovery<BackUpAction>
   double command_x_;
   double command_speed_;
   double simulate_ahead_time_;
+  double transform_tolerance_;
```

Comment:
You can remove the transform_tolerance_ from the individual plugins and instead add them to the `recovery.hpp` file to get and store them. Then the specific plugins will have them available for use and only define it once.

Reply:
Good idea, will update this.